lms/templates/sysadmin_dashboard.html

```diff
@@ -178,6 +181,31 @@ <h2 class="instructor-nav">
   <div class="form-actions">
 	<button type="submit" name="action" value="del_course">${_('Delete course from site')}</button>
   </div>
+  <hr />
+  <h4>Edit Course Tabs</h4>
+  <ul class="list-input">
+	  <li class="field text">
+		  <label for="course_id">
+			  ${_('Course ID')}:
+		  </label>
+		  <input type="text" name="course_id" style="width:60%"/>
+	  </li>
+  </ul>
+  <div class="form-actions">
+	<button type="submit" name="action" value="get_current_tabs">${_('Get Current Tabs')}</button>
+  </div>
+  <br />
```

Comment:
I'd rather we style this via CSS instead of manual line breaks; it makes for more maintainable code in the future.